docs/config/extras.md

```diff
@@ -63,30 +63,23 @@ Users of this flag should note that they may see an increase in their bundle siz
 
 Defaults to `false`.
 
-### lifecycleDOMEvents
+### experimentalScopedSlotChanges
 
-Dispatches component lifecycle events. By default these events are not dispatched, but by enabling this to `true` these events can be listened for on `window`. Mainly used for testing.
+This option updates runtime behavior for Stencil's support of slots in **scoped** components to match more closely with
+the native Shadow DOM behaviors.
 
-| Event Name                    | Description                                            |
-| ----------------------------- | ------------------------------------------------------ |
-| `stencil_componentWillLoad`   | Dispatched for each component's `componentWillLoad`.   |
-| `stencil_componentWillUpdate` | Dispatched for each component's `componentWillUpdate`. |
-| `stencil_componentWillRender` | Dispatched for each component's `componentWillRender`. |
-| `stencil_componentDidLoad`    | Dispatched for each component's `componentDidLoad`.    |
-| `stencil_componentDidUpdate`  | Dispatched for each component's `componentDidUpdate`.  |
-| `stencil_componentDidRender`  | Dispatched for each component's `componentDidRender`.  |
+When set to `true`, the following behaviors will be applied:
 
-### scopedSlotTextContentFix
-
-An experimental flag that when set to `true`, aligns the behavior of invoking the `textContent` getter/setter on a scoped component to act more like a component that uses the shadow DOM. Specifically, invoking `textContent` on a component will adhere to the return values described in [MDN's article on textContent](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#description). Defaults to `false`.
+<!-- TODO(NOW): confirm fix versions -->
```

Comment:
I'll make sure these versions are accurate before this merges, just depends how this lines up with upcoming releases